FirebaseRemoteConfig/Sources/Public/FirebaseRemoteConfig/FIRRemoteConfig.h

```diff
@@ -30,6 +30,10 @@ extern NSString *const _Nonnull FIRNamespaceGoogleMobilePlatform NS_SWIFT_NAME(
 extern NSString *const _Nonnull FIRRemoteConfigThrottledEndTimeInSecondsKey NS_SWIFT_NAME(
     RemoteConfigThrottledEndTimeInSecondsKey);
 
+extern @interface FIRConfigUpdateListenerRegistration : NSObject
```

Comment:
Please add a comment explain what it does, you can refer to firestore documentation as well for format similarity.

Reply:
Thanks! Added comments for the interface as well as the function. SImilar to Firestores'
https://github.com/firebase/firebase-ios-sdk/blob/78cc446bb68d34b7b54b1b8f6a62f37ad540fe83/Firestore/Source/Public/FirebaseFirestore/FIRListenerRegistration.h#L21